packages/node-utils/src/tests/findProcessFromIncomingPort.test.ts

```diff
@@ -19,10 +19,15 @@ describe('findProcessFromIncomingPort', () => {
             const processInfo = await findProcessFromIncomingPort(port);
             expect(processInfo).toBeDefined();
 
-            if (process.platform === 'win32') {
-                expect(processInfo?.name).toEqual('Node.js');
-            } else {
-                expect(processInfo?.name).toEqual('MainThread');
+            switch (process.platform) {
+                case 'win32':
+                    expect(processInfo?.name).toEqual('Node.js');
+                    break;
+                case 'darwin':
+                    expect(processInfo?.name).toEqual('node');
```

Comment:
it could have some runtime consequences where we rely on this value maybe? 🤔 I haven't checked, just a thought

Reply:
No, I am just stupid and somehow ran it on v22 🤦 . Closing as invalid.

Reply:
OK I am even more stupid 🤦 . It is still `node` on v24. 
<img width="1418" height="648" alt="Screenshot 2025-12-05 at 10 36 46" src="https://github.com/user-attachments/assets/5b299bae-4022-4b66-b95f-c3cca988acec" />